sigmastate/src/main/scala/sigmastate/types.scala

```diff
@@ -1361,7 +1369,8 @@ object SOption extends STypeCompanion {
     FoldMethod,
     */
     MapMethod,
-    FilterMethod
+    FilterMethod,
+    IsEmptyMethod
```

Comment:
we cannot add new methods to ErgoTree, because it will break the protocols.

Reply:
Removed.